src/DotNetWorker.Grpc/protobuf/src/proto/FunctionRpc.proto

```diff
@@ -273,6 +281,12 @@ message RpcFunctionMetadata {
 
   // Raw binding info
   repeated string raw_bindings = 10;
+
+  // unique function identifier (avoid name collisions, facilitate reload case)
+  string function_id = 13;
+
+  // A flag indicating if managed dependency is enabled or not
+  bool managed_dependency_enabled = 14;
```

Comment:
This is unrelated to single load request change. Could you remind me what this one is for?

Reply:
From [this commit](https://github.com/Azure/azure-functions-language-worker-protobuf/commit/67ecc31fcf4ee44bd60c45d65fa1c854b8723ea8) in the protobuf repo ~21 hrs ago. I'm not familiar with the change, but I'm pretty sure it's Stein related.

Reply:
Oh it is - it was part of some refactoring done to better work with legacy apps. These fields were moved from a different message to this one.

Reply:
This is part of host fallback workitem. There were inconsistencies in host and protobuf repo. Yesterday, all these PRs fully resolved them.